templates/JSResolverOCHTTPSTemplate.mjs

```diff
@@ -1048,4 +1047,5 @@ function resolveGraphDBQuery(query) {
 }
 
 
-module.exports = { resolveGraphDBQueryFromAppSyncEvent, resolveGraphDBQueryFromApolloQueryEvent, resolveGraphDBQuery, refactorGremlinqueryOutput };
+export default { resolveGraphDBQueryFromAppSyncEvent, resolveGraphDBQueryFromApolloQueryEvent };
```

Comment:
Is there a reason to split these into 2 export statements?

Reply:
It's because I had to change the import to
```
import pkg from './output.resolver.graphql.js';
const {resolveGraphDBQueryFromAppSyncEvent, refactorGremlinqueryOutput} = pkg;
```
for it to work with appsync, and for that to work, I had to change the exports for `resolveGraphDBQueryFromAppSyncEvent` and `resolveGraphDBQueryFromApolloQueryEvent` to export default. The other exports didn't work with export default so that's why there's 2 export statements.

Reply:
Should work without default:
```
export { resolveGraphDBQueryFromAppSyncEvent, resolveGraphDBQueryFromApolloQueryEvent };
```